pygeos/creation.py

```diff
@@ -175,7 +175,7 @@ def geometrycollections(geometries):
 
 
 def prepare(geometry, **kwargs):
-    """Compute a prepared geometry.
+    """Add a prepared geometry to a geometry, improving performance of other operations.
```

Comment:
```suggestion
    """Prepare a geometry, improving performance of other operations.
```

I find "adding" a bit strange, since it's an implementation detail that it is added to the C struct?